src/analysis/function_parameters.rs

```diff
@@ -198,12 +198,36 @@ pub fn analyze(
         .filter_map(|p| p.array_length.map(|pos| (pos, p)))
         .collect();
 
+    let mut to_remove = Vec::new();
+    let mut correction_instance = 0;
+
+    for par in function_parameters.iter() {
+        if par.scope.is_none() {
+            continue;
+        }
+        if let Some(index) = par.closure {
+            to_remove.push(index);
```

Comment:
You could take a look at how this is handled for `array` attributes. See e.g. `find_index_to_ignore` in `src/analysis/functions.rs`. Not sure if that improves anything though